spec/factories/activity.rb

```diff
@@ -4,7 +4,7 @@ module SupplejackApi
   FactoryBot.define do
     factory :activity, class: SiteActivity do
       created_at 1.day.ago.utc
-      updated_at Date.today
+      updated_at Time.now.utc.to_date
```

Comment:
I mean I don't think it matters really, but `_at` would imply you want a time, not a date? While `_on` fields are typically dates.

So you could simply `Time.now.utc`

Totally doesn't matter, `ActiveRecord` will convert it to a `DateTime` anyways 👍

Reply:
This I think is the only `_at` field using `to_date` — the others are using `.midday`.
Just commenting it cause it's interesting thinking about all the date/time stuff haha.